Makefile

```diff
@@ -149,16 +149,15 @@ ifneq (${BUILD_PLATFORM},aarch64)
 endif
 
 test:
-	${GOTEST} -tags unit -coverprofile cover.out -timeout=60s ./agent/...
+	cd agent && GO111MODULE=on ${GOTEST} ${VERBOSE} -tags unit -mod vendor -coverprofile ../cover.out -timeout=60s ./... && cd ..
```

Comment:
updating to latest go version in the modules file requires us to fully buy-in to modules here, so that is why these make targets are being updated.